app/models/review.rb

```diff
@@ -12,4 +13,12 @@ class Review < ActiveRecord::Base
   def self.search(search)
     where('description ILIKE ?', "%#{search}%")
   end
+
+  def total_score
```

Comment:
Great method for your review model here. Can you add unit specs to test just this method?